prisma/seed.ts

```diff
@@ -91,6 +137,7 @@ function chooseFromN<T>(choices: Readonly<T[]>, n: number): T[] {
  * @param end the end of the range, inclusive
  * @returns a list of numbers [start, ..., end]
  */
+// eslint-disable-next-line
```

Comment:
add a todo comment to grab attention to this (so it doesn't get buried in the codebase)